coderd/httpmw/apikey.go

```diff
@@ -51,6 +51,8 @@ type OAuth2Configs struct {
 	Github OAuth2Config
 }
 
+const loggedOutErrorMessage string = "You are logged out. Please log in to continue."
```

Comment:
"You are logged out or your session has expired. Please log in again to continue."

Reply:
Much better. Thank you!